config/environments/test.rb

```diff
@@ -43,14 +28,19 @@
   # Disable request forgery protection in test environment.
   config.action_controller.allow_forgery_protection = false
 
+  # Store uploaded files on the local file system in a temporary directory
+  config.active_storage.service = :test
+
+  config.action_mailer.perform_caching = false
+
   # Tell Action Mailer not to deliver emails to the real world.
   # The :test delivery method accumulates sent emails in the
   # ActionMailer::Base.deliveries array.
   config.action_mailer.delivery_method = :test
   config.action_mailer.default_url_options = { host: 'localhost', port: '3000' }
 
   # Randomize the order test cases are executed.
-  config.active_support.test_order = :random
+  config.active_support.test_order = :sorted
```

Comment:
to be random or not to be random, is that the question? - why was this changed?

Reply:
Strictly for easily repeating the same test sequence after fixing a problem.  This could go be set back to random on this point.  Let me know if you want me to make that change and I'll put it in.

Reply:
Thought about this some more and decided to put it back to random.  It is committed.